src/plugins/auto/auto_schedule.cpp

```diff
@@ -143,11 +143,13 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
                                                                            _loadContext[ACTUALDEVICE].networkPrecision,
                                                                            _autoSContext->_modelPriority);
     }
+    if (!_autoSContext->_startupfallback)
+        _loadContext[ACTUALDEVICE].deviceInfo.config[ov::intel_auto::enable_startup_fallback.name()] = InferenceEngine::PluginConfigParams::NO;
```

Comment:
looks like we don't need to carry on this config? do we?

Reply:
Yes, removed it.